Stream module is broken, nginx doesn't start worker processes when
the module is loaded.

--- a/config
+++ b/config
@@ -43,7 +43,7 @@
     ngx_module_srcs="$ngx_module_srcs $HTTP_ACCOUNTING_SRCS"
 fi
 
-if [ $STREAM != NO ]
+if false # [ $STREAM != NO ]
 then
     ngx_module_type=STREAM
     if test -n "$ngx_module_name"; then
